wireless/bluetooth/nimble/Makefile

```diff
@@ -20,18 +20,24 @@
 
 include $(APPDIR)/Make.defs
 
-PRIORITY  = 255
-STACKSIZE = $(CONFIG_NIMBLE_STACKSIZE)
-
 NIMBLE_UNPACKDIR = mynewt-nimble
 NIMBLE_ROOT = $(APPDIR)/wireless/bluetooth/nimble/$(NIMBLE_UNPACKDIR)
-
--include $(NIMBLE_ROOT)/porting/examples/nuttx/Make.defs
-
 CONFIG_NIMBLE_REF := $(patsubst "%",%,$(strip $(CONFIG_NIMBLE_REF)))
 NIMBLE_TAR := $(CONFIG_NIMBLE_REF).tar.gz
 NIMBLE_URL := https://github.com/apache/mynewt-nimble/archive/$(NIMBLE_TAR)
 
+ifneq ($(CONFIG_NIMBLE_PORTING_EXAMPLE),)
+# NimBLE porting example built-in application
+
+-include $(NIMBLE_ROOT)/porting/examples/nuttx/Make.defs
```

Comment:
@raiden00pl I'm having an issue with getting nimBLE running with this patch. Actually the nimBLE app is running, but I see that changing newly added configuration options does not bring any effect. In other words `wireless/bluetooth/nimble/include/syscfg/syscfg.h` and `wireless/bluetooth/nimble/include/logcfg/logcfg.h` are not take into account.
After I'm doing a hack to get those files picked by the build system the nimBLE build system find it incomplete. For example trying to build Linux version of some files instead of NuttX

Reply:
I just figured out that I do not have `EXAMPLES_NIMBLE` enabled

Reply:
Now example from the NimBLE repo is selected as default (CONFIG_NIMBLE_PORTING_EXAMPLE=y).
But I think that the example from our repo should be the default.
I didn't change it in this PR and left it for later, as it required a synchronous change in boards configuration.

Reply:
@raiden00pl I have some question about nimBLE and tried to send you an e-mail, but no response till now. Could you please e-mail me or maybe we can talk in LinkedIn?

Reply:
@pkarashchenko Wow. I see now that protonmail has stopped sending me notifications of new messages in my inbox. I see your message now and I'll reply when I find a moment.